apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { device, expect } from "detox";
```

Comment:
Do you need the device import?

Reply:
I think i do? at least i'm getting some error on VSC if i remove it

Reply:
It's not being used anywhere in this file though. Try removing it and rerunning the test